velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -97,7 +97,12 @@ class StringFunctionsTest : public FunctionBaseTest {
 
     int refCountAfterEval =
         bufferRefCounts(stringVector->asFlatVector<StringView>());
-    EXPECT_EQ(refCountAfterEval, 2 * refCountBeforeEval) << "at " << query;
+
+    if (query.find("substring(") != std::string::npos) {
```

Comment:
Why is this behavior different ?

Reply:
We added the test case for substring alias, there are more calls compared with substr only.   I made the changes so that buffer reference count works for both of substr and substring.